src/compiler/checker.ts

```diff
@@ -3682,7 +3682,8 @@ namespace ts {
                     tracker: tracker && tracker.trackSymbol ? tracker : { trackSymbol: noop, moduleResolverHost: flags! & NodeBuilderFlags.DoNotIncludeSymbolChain ? {
                         getCommonSourceDirectory: (host as Program).getCommonSourceDirectory ? () => (host as Program).getCommonSourceDirectory() : () => "",
                         getSourceFiles: () => host.getSourceFiles(),
-                        getCurrentDirectory: host.getCurrentDirectory && (() => host.getCurrentDirectory!())
+                        getCurrentDirectory: host.getCurrentDirectory && (() => host.getCurrentDirectory!()),
+                        getProbableSymlinks: host.getProbableSymlinks,

```

Comment:
I think you want to use `mayBeBind` here instead so if host is a class we are not loosing this context ?

Reply:
Oh, good catch, thank you. Also wasn’t aware of that utility 👍